README.md

```diff
@@ -7,35 +8,31 @@ This is a development version of the new talis-php version. It is not yet intend
 This early version pulls existing individual client libraries into one single library with minimal
 changes.
 
-Before releasing for general usage there will be major changes to the API. The library will move 
+Before releasing for general usage there will be major changes to the API. The library will move
 away from the use of internal Talis project names like Persona, Critic, Babel and Echo etc.
 Instead it will use more externally relavant names like ```ListReviews``` and ```Files```.
 The API will also move to a more domain driven design rather than the service driven design
 of the individual libraries
 
 See issue: https://github.com/talis/talis-php/issues/2 and Milestone: https://github.com/talis/talis-php/milestone/1
 
-
 ## Contributing
 
 A Dockerfile is provided to make it easy to get a local development environment
 up and running to develop and test changes. Follow these steps:
 
-```bash
-
 # Build the development image
 
+```bash
 git clone https://github.com/talis/talis-php.git
 cd talis-php
-docker build -t "talis-php:dev" --build-arg git_oauth_token=<your github oauth token> --build-arg persona_oauth_client=<your oauth client> --build-arg persona_oauth_secret=<your oauth client secret> .
-
-# When the above has built successfully you can run and connect to the container
-docker run -v /path/on/host/machine/to/talis-php:/var/talis-php -i -t echo-php-client:dev /bin/bash
+ant build
+```
 
-# Then inside the container
+# When the above has built you can run the tests
 
-service redis-server start
-ant init
-ant test
+```bash
+docker-compose run test
```

Comment:
to make it easier to run the tests I have included a docker-compose file which also follows the same design concept are the other primitives